Gaijinizer patch/Map045.txt

```diff
@@ -33,9 +33,9 @@ Wait a sec. I'm getting your exact coordinates.#Aesthetic
 俺の部隊が設置した\\C[2]灯火装置を頼りに進んでいけ。
 > CONTEXT: Map045/events/4/pages/0/29/Dialogue/yuuma
 > CONTEXT: Map183/events/4/pages/0/29/Dialogue/yuuma
-To get to the harbor, you'll need to walk around downtown for a bit. 
-You can rely on the \\C[2] lighting equipment that my troops
-set up.#Aesthetic
+To get to the harbor, you'll need to walk around 
+downtown for a bit. You can rely on the \\C[2]
+lighting equipment that my troops set up.#Aesthetic
```

Comment:
This is hardly major, but I feel we could use a better sounding set of words then "lighting equipment", even though that's technically what they are. Might need to go through Chapter 2 myself to double check what he's referring to.

Reply:
![Lighting equipment](https://user-images.githubusercontent.com/46212742/65481783-d2b06580-deea-11e9-80c2-e1f33c40ef84.png)

I'm guessing he's referring to these things I've circled in the image which I'm assuming are supposed to guide the player. The literal translation would be something along the lines of "Lamp Device" or "Light Equipment", but you'd probably be able to phrase it better so it flows better.